src/test/java/com/zerobank/pages/BasePage.java

```diff
@@ -27,7 +27,25 @@ public void navigateToModule(String Tab){
     }
 }
 
+    //I added because I need to navigate to the tab as well
```

Comment:
imo it would be better if we overrite navigateToModule so i just want you to delete "s" and update pull request. Thx